src/App.js

```diff
@@ -1,10 +1,21 @@
+import { Route, BrowserRouter, Routes } from 'react-router-dom';
 import React from 'react';
 
```

Comment:
- [ ] _**I am using this place to suggest some changes which are not related to this App.js file.**_  I recommend deleting all boilerplate (texts, images, etc.) from the `public` folder. Because they make your codebase unclean.
- [ ] _**In your Readme file**_ to make it more informative I suggest doing the followings:
- Add a project screenshot;
- Add a GitHub repo issues page link;
- Add a license;